src/Controller/MenuController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\DoStuff;
+use App\Util\Debugger;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Service\CategorieService;
+use App\Entity\Produit;
+use App\Entity\Menu;
+
+class MenuController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        /** @var Debugger $debugger */
+        $debugger = $container->get('debug');
+
+        $entityManager = $container->get('entity.manager');
+        $entityManager->resetDatabase();
+
+        ob_start();
+        $menu = new Menu();
+
+        echo "<h1>Créer un menu</h1> <br>";
```

Comment:
careful, `<br>` is a self closing HTML node. it should be `<br/>`
since you do not use any variable inside your string, using " double quotes cost more thant simple ones '.